src/main/java/ui/UiRenderer.java

```diff
@@ -233,22 +233,22 @@ private static List<String> wrapText(String text, int maxLengthToOccupy) {
         }
         return lines;
     }
-    
+
 
     /**
      * Prints the task for the day.
-     * 
+     *
      * @param dayTasks The list of tasks for the day.
      * @param taskIndex The index of the task to print.
      */
     public static void printTaskForDay(List<Task> dayTasks, int taskIndex) {
         if (taskIndex < dayTasks.size()) {
             Task task = dayTasks.get(taskIndex);
             String taskDescription = task.getName();
-            String displayString = 
-                    (taskIndex + 1) + "." + 
-                    task.getDisplayFormat() +
-                    taskDescription;
+            String displayString =
+                    (taskIndex + 1) + "." +
+                            task.getDisplayFormat() +
+                            taskDescription;
```

Comment:
Similar to the comment above!

Reply:
Nice catch there! I had missed it since it had passed the checkstyle tests.